src/components/middle/message/Message.tsx

```diff
@@ -730,8 +734,21 @@ const Message: FC<OwnProps & StateProps> = ({
     replyStory,
   );
 
+  const WITH_BOTTOM_ELEMENT_GAP = 5.625 * REM;
+  const SPONSORED_MESSAGE_GAP = 10 * REM;
```

Comment:
@korenskoy since the maximum length for a sponsored message is 160 characters + title + button or other variants, the height will be ~160px

![sponsored-message](https://github.com/Ajaxy/telegram-tt/assets/10471508/95b3d16c-cdce-4492-8a76-ae9c077f48a3)

in order to cover edge cases I'll increase it to 11 * REM,  to get extra gap and only 1 static value, just to not recalculate it dynamically by any element API that will trigger reflow:
https://gist.github.com/paulirish/5d52fb081b3570c81e3a